Gemfile

```diff
@@ -27,14 +28,19 @@ gem 'sdoc', '~> 0.4.0',          group: :doc
 gem 'spring',        group: :development
 
 # Use ActiveModel has_secure_password
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use unicorn as the app server
-# gem 'unicorn'
+gem 'bcrypt', '~> 3.1.7'
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
 # Use debugger
 # gem 'debugger', group: [:development, :test]
-
+gem 'rspec-rails', group: [:development, :test]
+gem 'simplecov', group: [:development, :test]
+gem 'rails_12factor'
+
+group :production do
+  gem 'unicorn'
```

Comment:
You've got unicorn in your production group, but no Procfile to run it. Are you sure that Heroku's running unicorn?